exercises/practice/leap/leap.odin

```
@@ -0,0 +1,5 @@
+package leap
+
+is_leap_year :: proc(year: int) -> bool {
+	return false
```

False is a potentially valid answer for some inputs. Can we raise an error that this code needs to be implemented?

---

Fixed by adding an error result in the return values.

---

I also changed this to a compile-time panic, as per @ErikSchierboom's suggestion. I think this is much cleaner and more helpful to the student.

---

Yeah, that's the approach the other compiled language tracks like C# take (see https://github.com/exercism/csharp/blob/main/exercises/practice/leap/Leap.cs).